third_party/rbs_parser/BUILD

```diff
@@ -0,0 +1,37 @@
+load("@rules_ragel//ragel:ragel.bzl", "ragel")
```

Comment:
The code actually comes from https://github.com/Shopify/rbs_parser which provides `rbs2rbi`, a standalone tool to translate RBS files to RBI files.

Instead of rewriting it for Sorbet I followed the same approach already used to import the `third_part/parser`. The [first commit](5fbce460dc19c22b39aa6b019a1185f9e58e4c87) introduce a script to import the sources.

I'm open to import them somewhere else if it makes more sense.